code/game/objects/items/apc_frame.dm

```diff
@@ -6,8 +6,8 @@
 	item_state = "syringe_kit"
 	w_class = WEIGHT_CLASS_SMALL
 	var/result_path
-	var/inverse = 0
-	// For inverse dir frames like light fixtures.
+	var/inverse = 0 // For inverse dir frames like light fixtures.
+	var/onto_wall = FALSE //If the object places onto the wall instead of the floor
```

Comment:
is this correct? surely not.
If they were truly on the walls, then you would be able to use them on both sides, I'm pretty sure they're just pixel shifted like every other wall item.